### PR TITLE
Added basic automoderator capabilities

### PR DESCRIPTION
**Pull request by BrycenW**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -48,9 +48,9 @@ def count_all_likes(token,group_id):
 
 
 last_num = 0
-token = "dbce80c042ef0133562d05f0d49317f6"#raw_input("Input token: ")
-bot_id = "7e819111ff8f330b299db0679f"#raw_input("Input bot_id: ")
-group_id = "15781720"#raw_input("Input group_id: "
```
Comment (CodaFi) [inline bot.py]: Thank god you finally removed this.

Comment (BrycenW) [inline bot.py]: I figured it was time

File: automod.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+		if int(message['count']) > last_num:
+			print message
+			print message['user_id'] + message['name']
+			if str(message['user_id']) == str(user_id) and nickname != message['name']:
+				rename_member(bot_id, user_id, token, group_id, nickname)
+			last_num = message['count']
+		else:
+			test()
+			time.sleep(1)
+			last_num = int(message['count'])
```
Comment (CodaFi) [inline automod.py]: Newline here please.

**Comment by CodaFi** [comment]
Can the diagnostic `print`s be removed before this is merged?

File: automod.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import time
+from inOut import *
+
+token = raw_input("Input token: ")
+bot_id = raw_input("Input bot_id: ")
+group_id = raw_input("Input group_id: ")
+
+def check_mess(text):
+	print text
+	if "kick me" in text.lower():
```
Comment (CodaFi) [inline automod.py]: Without the `print`s this is just

```python
def check_mess:
  return "kick me" in text.lower()
```

Comment (CodaFi) [inline automod.py]: Would also help to have a more descriptive name in line with Python-isms.  Possibly `contains_kickme` or `has_kickable_text`.

Comment (BrycenW) [inline automod.py]: It's defined this way to be easy to change the moderating parameters.

Comment (CodaFi) [inline automod.py]: I know, I respect the abstraction aspect, it's just that the semantics of this are not immediately obvious from the name.

File: automod.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+			last_num = message['count']
+		else:
+			test()
+			time.sleep(1)
+			last_num = int(message['count'])
+
+def rerename(user_id, nickname):
+	rename_member(bot_id, user_id, token, group_id, nickname)
+	last_num = 0
+	while 1:
```
Comment (CodaFi) [inline automod.py]: If I'm not mistaken, this will spin forever.

File: automod.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+	print text
+	if "kick me" in text.lower():
+		print "true"
+		return True
+	print "false"
+	return False
+
+def moderate():
+	last_num = 0
+	while 1:
```
Comment (CodaFi) [inline automod.py]: This one, too, should diverge.

File: automod.py (hunk 2 above)
Comment (BrycenW) [inline automod.py]: Where?

File: automod.py (hunk 4 above)
Comment (BrycenW) [inline automod.py]: That is correct

File: automod.py (hunk 2 above)
Comment (CodaFi) [inline automod.py]: Hm, it moved my comment on the diff.  This was supposed to be at the end of the file.

**Pull request merged**